src/reportportal/rp_query.py

```
@@ -487,6 +488,97 @@ def _resolve_test_target(client: 'ReportPortalAPIClient', launch_id: str, target
+            for i, attempt in enumerate(chain, 1):
+                status = attempt.get('status', 'N/A')
+                print(f"  Attempt {i} ({status}):")
+                if status == utils.STATUS_FAILED:
+                    logs = _fetch_error_logs(attempt.get('id'), client)
+                    _print_logs(logs, indent="    ")
+            print()
+        else:
+            logs = _fetch_error_logs(item.get('id'), client)
+            print(f"{item_name}:")
+            if not logs:
+                print(f"  (no error logs found)")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)